src/util/common.js

```diff
@@ -74,22 +74,24 @@ export function getUserType() {
   return token ? token.substr(token.length - 1) : null
 }
 
-export async function getTokenFB() {
-  window.open(
+export async function getUserFbInfo() {
+  let popup = window.open(
```

Comment:
&scope=email&response_type=token,granted_scopes&  -> those query are for ? can remove ?